### PR TITLE
Bugfix/issue 248/migrate after deploy for google auth

### PR DESCRIPTION
**Pull request by maxgrove42**
# #248

## Description

- Adds a after_deploy script to migrate after deployment. This should support the Google Auth issues

## Change Type (delete non-relevant options)

- 🪲 Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

**Review by QahtanAlJammali — APPROVED** [review]
Excited about the google authentication fix!  thank you

File: src/static/js/manage_organization.js [hunk 1]
```diff
@@ -1,3 +1,19 @@
+// Function to confirm deletion of an organization
+function confirmDelete() {
+    if (confirm("Are you sure you want to delete this organization?")) {
```
Comment (arsalananwar11) [inline src/static/js/manage_organization.js:3]: I think this file has been updated. Can you please take the latest?

**Review by arsalananwar11 — CHANGES_REQUESTED** [review]

File: src/static/js/manage_organization.js (hunk 1 above)
Comment (maxgrove42) [reply src/static/js/manage_organization.js:3]: weird, I didn't change these files, so I'll merge in from develop and just take develop branch as is @arsalananwar11

**Comment by maxgrove42** [comment]
@arsalananwar11 merged in from develop - i think you have to provide approval !

**Review by arsalananwar11 — APPROVED** [review]
Looks great, thanks!

**Pull request merged**